browser/speedreader/speedreader_browsertest.cc

```diff
@@ -8,6 +8,7 @@
 #include "brave/app/brave_command_ids.h"
 #include "brave/common/brave_paths.h"
 #include "brave/components/speedreader/speedreader_switches.h"
+#include "brave/components/speedreader/features.h"
```

Comment:
sorting

Reply:
make sure to run `npm run lint` or ci will fail anyway

Reply:
i think linter doesn't catch this...

Reply:
that's what @darkdh said too, but it used to catch these. I wonder what changed?

Reply:
github interface not showing correctly, but should be fixed now